templates/common/debug-info.html.jinja

```diff
@@ -0,0 +1,8 @@
+<details class="render-info">
```

Comment:
I'm not sure I would call this `debug-info`. These values ultimately got surfaced because folks wanted to see them in normal use.